configuration/multimedia.md

```diff
@@ -386,3 +384,109 @@ listenAndAnswer(null, null)
 listenAndAnswer("javasound", "sonos:PLAY5:kitchen")
 listenAndAnswer("voskstt", "voicerss", "voicerss:frFR_Axel", "system,rulehli", "javasound", "sonos:PLAY5:kitchen", "fr-FR", "listeningItem")
 ```
+
+### The Built-in Interpreter
+
+These interpreter is available by default and includes a build-it grammar for English, German, French and Spanish that you can extend using the voiceSystem metadata. These are some examples of the built-in English grammar:
```

Comment:
The interpreter is available by default and includes built-in grammar ... that can be extended using ...

Here are some examples ...